preset/README.md

```diff
@@ -0,0 +1,41 @@
+# Babel Preset for Source Code
+
+> A Babel 6 preset that compiles official ES2015+ down to ES5 by automatically determining the Babel plugins and polyfills you need based Salesforce's supported browsers. This includes the Design System for React project.
```

Comment:
A Babel 6 preset that makes DSR compatible with Salesforce’s supported browsers. This enables your module bundler (ie: Webpack) to transpile DSR. Using this will make it easier to upgrade in the future without having to manually reconfigure your Babel settings to be compatible with new language features DSR may use.